groups/middleware/authorization.go

```diff
@@ -602,3 +702,22 @@ func (am *authorizationMiddleware) extAuthorize(ctx context.Context, extOp svcut
 
 	return nil
 }
+
+func (am *authorizationMiddleware) Callout(ctx context.Context, session authn.Session, op string, params map[string]interface{}) error {
+	pl := map[string]interface{}{
+		"entity_type": policies.GroupType,
+		"sender":      session.UserID,
```

Comment:
Let's rename to `subject_id` - it can be user or client.

Reply:
@dborovcanin  I think we can have `user_id`, because we don't have callout for HTTPs 
Then if we have `subject_id` , then we need mention the `subject_type`